hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ReconTaskControllerImpl.java

```diff
@@ -178,12 +193,33 @@ public synchronized void reInitializeTasks(ReconOMMetadataManager omMetadataMana
     }
     ReconConstants.resetTableTruncatedFlags();
 
+    ReconTaskStatusUpdater reprocessTaskStatus = taskStatusUpdaterManager.getTaskStatusUpdater(REPROCESS_STAGING);
```

Comment:
I think, we need to rename this task or step for STAGING DB switchover success/failed because REPROCESS step for tasks will be success/failed on individual level, and that we need to still do at each task level as done currently already in .`thenAccept` block. so for STAGING DB switchover success/failed, we need to just track.

Reply:
updated